artifacts/deploy/karmada-search.yaml

```diff
@@ -77,7 +77,8 @@ metadata:
     apiserver: "true"
 spec:
   ports:
-    - port: 443
+    - appProtocol: https
```

Comment:
Is this also useful when working with other system other than Istio? In other words, if another system requires another appProtocol?
(I'm not an expert in this area, maybe stupid question)

Reply:
Also not an expert as we only use Istio :)

Reply:
> In other words, if another system requires another appProtocol? (I'm not an expert in this area, maybe stupid question)

I think the configuration files we provide here should be universal and independent of specific usage. Can we use them as a practice to adapt Karmada to Istio and showcase it in tutorials or other documents?